effekt/shared/src/main/scala/effekt/source/Tree.scala

```diff
@@ -209,6 +250,73 @@ enum Param extends Definition {
 }
 export Param.*
 
+/**
+ * A `List` (usually of `Tree`s) with a `span` that spans all elements of the list
+ */
+case class Many[T](list: List[T], span: Span) {
+  def unspan: List[T] = list
```

Comment:
I don't quite understand why the explicit `unspan` as a separate method as opposed to `list`.
I would have imagined directly
```suggestion
case class Many[T](unspan: List[T], span: Span) {
```